design/pipeline-artifacts.md

```diff
@@ -59,19 +59,25 @@ Listing just the unique properties of the `download` step:
   path: string - the destination path for where to download the artifact(s)
 ```
 
-### Automated download
+### Automated download behavior - job vs deployment
 
-The "magic" of download is how often you _don't_ have to specify it.
-By default, all artifacts produced in the current pipeline and all artifacts in all referenced pipelines will be downloaded.
-As soon as you include an explicit `download` step (or task), the user is in total control - no more automatic download for that job.
-Similarly, you can leave out `artifact`, `patterns`, or `path` to get some automatic behaviors (described below).
+Usually a 'job' publishes build artifacts which are deployed to an environment as part of the 'deployment' job.
+Therefore, the automatic artifact download behavior varies between a 'job' and 'deployment'.
+
+#### job
+Artifacts are not automatically downloaded in regular jobs. However, if you require artifacts to be downloaded in your jobs, you need to explicitly use `download`.
```

Comment:
Just to be clear, I would reword the second sentence: "To download artifacts in these jobs, use the `download` task.`

Reply:
Updated as suggested.